### PR TITLE
Rename "compress" terminology to "artifacts" to be more representative

### PR DESCRIPTION
**Pull request by develohpanda**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -87,15 +87,14 @@ jobs:
           MACOS_CERTIFICATE_PWD: ${{ secrets.DESIGNER_MAC_CSC_KEY_PASSWORD }}
           PKG_NAME: ${{ steps.inso-variables.outputs.pkg-name }}
           BUNDLE_ID: ${{ steps.inso-variables.outputs.bundle-id }}
-          ARTIFACT_LOCATION: compressed
           VERSION: ${{ steps.inso-variables.outputs.inso-version }}
 
-      # We don't need to notarize and staple on every commit
+      # # We don't need to notarize and staple on every commit
       # - name: Notarize installer package
```
Comment (dimitropoulos) [inline .github/workflows/test.yml:93]: If this fix related to this PR?

Comment (develohpanda) [reply .github/workflows/test.yml:93]: How so? I didn't re-enable the steps, they should be disabled as per #4082, but I just updated the paths within the commented steps. So, to that end, yes the changes in these commented lines are related to this PR.

Comment (dimitropoulos) [reply .github/workflows/test.yml:93]: I was only asking about the one line I selected, it's no biggie, was just wondering where it came from.

Comment (develohpanda) [reply .github/workflows/test.yml:93]: Just a bad merge while resolving conflicts

Comment (develohpanda) [reply .github/workflows/test.yml:93]: Reverted in bc60bc0eded3a6d70caf634d055f9133907b4221

Comment (dimitropoulos) [reply .github/workflows/test.yml:93]: cool cool, thanks :)

**Review by johnwchadwick — APPROVED** [review]

**Pull request merged**